html/index.html

```diff
@@ -4,15 +4,56 @@
 <head>
     <title>Cacophonator Setup</title>
     <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="viewport" content="max-width=device-max-width, initial-scale=1">
     <link rel="stylesheet" href="/static/css/bootstrap.min.css">
 </head>
 
 <body>
     {{template "navbar"}}
+
+    <div class="container">
+            <div class="row">
+                <div class="col-xs-12 w-100" >
+                    <a class="btn btn-primary btn-block mt-3" style="max-width: 400px;" href="/camera" role="button">Camera</a>
```

Comment:
It would be better to have a CSS class in a `<style>` section at the top which sets max-width and then use that for the buttons.